src/Microsoft.AspNet.Mvc.Core/Infrastructure/MvcRouteHandler.cs

```diff
@@ -87,7 +87,7 @@ public VirtualPathData GetVirtualPath(VirtualPathContext context)
                 {
                     _logger.ExecutingAction(actionDescriptor);
 
-                    var startTime = Environment.TickCount;
+                    var startTime = DateTime.UtcNow;
```

Comment:
This was addressed as part of https://github.com/aspnet/Mvc/commit/c125b4e59badf551c85bbc8d6ae8cb133fe6de7c